manifests/w/WHTA/Tamara/2023.0.8.9/WHTA.Tamara.installer.yaml

```
@@ -0,0 +1,39 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: WHTA.Tamara
+PackageVersion: 2023.0.8.9
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: -mode silent
+  SilentWithProgress: -mode silent
+  Custom: -mode silent -agreeToLicense yes

```

```suggestion
  Custom: -agreeToLicense yes
```
Custom represents the args that are passed in addition to the Silent / SilentWithProgress args.

This means with the current manifest, the following args will be passed:

`./installer.exe -mode silent -mode silent -agreeToLicense yes`

We can remove `-mode silent` from `Custom:` to get rid of the duplication